### PR TITLE
Redux toolkit

### PR DESCRIPTION
**Pull request by YuliaOvchinnikova**

### CODE REVIEW

File: client/src/ConfiguratorPage/ConfiguratorPage.js [hunk 1]
```diff
@@ -40,15 +40,17 @@ export const ConfiguratorPage = () => {
     },
   });
 
-  if (isLoading) {
+  if (status === "loading") {
```
Comment (nkabardin) [inline client/src/ConfiguratorPage/ConfiguratorPage.js:43]: try to always use constants for things like that.

something like status === IngredientStatus.LOADING

in typescript you would make IngredientStatus — enum. 

in plain javascript, you can do something like:

const IngredientStatus = {
   LOADING: 'loading'
}

File: client/src/PaymentPage/PaymentPage.js [hunk 2]
```diff
@@ -29,39 +32,38 @@ export const getCardType = (firstCharacter) => {
...
+  const status = useSelector(getOrderStatus);
 
   if (!pizza) {
     return <Redirect to="/" />;
   }
-  if (orderState === "sending") {
+  if (status === "sending") {
     return <p>Sending</p>;
-  } else if (orderState === "success") {
+  } else if (status === "success") {
```
Comment (nkabardin) [inline client/src/PaymentPage/PaymentPage.js:51]: OrderStatus.SUCCESS

**Review by nkabardin — APPROVED** [review]
Great job, some comments, but mostly its perfect. Check out the video review :)

**Comment by YuliaOvchinnikova** [comment]
I will fix comments as follow up PR

**Pull request merged**